client/settings/express-checkout-settings/woopay-preview.js

```diff
@@ -107,7 +108,36 @@ const CheckoutButton = ( { height } ) => {
 
 export default ( { storeName, storeLogo, customMessage, ...props } ) => {
 	const { style, ...restProps } = props;
-	const trimmedCustomMessage = ( customMessage || '' ).trim();
+
+	const preparedCustomMessage = useMemo( () => {
```

Comment:
We may want to tweak this function because HTML elements are not being escaped in the preview:

<img width="692" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/10233985/5dba965a-7844-48f8-9f06-f40d8628ac3c">

Though, they're being escaped at the WooPay checkout 👍 

<img width="370" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/10233985/6df75a86-45d3-4e8c-9058-0e3362f3a9a3">

Reply:
We discussed this over Slack and agreed that these elements are being escaped, but in order to match the actual implementation, they should not appear in the preview at all. I agree that the preview should look as similar to the implementation as possible. Fixed in [0194e9e](https://github.com/Automattic/woocommerce-payments/pull/7601/commits/0194e9ef2e2266e61310585b25b8d9a107d13fa4) and [7fce67e](https://github.com/Automattic/woocommerce-payments/pull/7601/commits/7fce67ef64e5979e14180de2ef86c5ea1387874b).